### PR TITLE
Allow using limited/unlimited termserver via config

### PR DESCRIPTION
**Pull request by makyo**

### CODE REVIEW

**Review by frankban — APPROVED** [review]
Looks good! So a follow up is planned for handling a config change after deployment?

**Comment by makyo** [comment]
@frankban config change works. If you update the `limit-termserver` key, it updates the termserver profile: https://pastebin.ubuntu.com/26216543/

File: reactive/jujushell.py [hunk 1]
```diff
@@ -78,7 +78,10 @@ def install_jujushell():
 def install_termserver():
     hookenv.status_set('maintenance', 'fetching termserver')
     try:
-        jujushell.save_resource('termserver', jujushell.termserver_path())
+        jujushell.save_resource(
+            'termserver', jujushell.termserver_path())
+        jujushell.save_resource(
+            'limited-termserver', jujushell.termserver_path(limited=True))
```
Comment (jcsackett) [inline reactive/jujushell.py]: `save_resource` pulls the resource down from the charmstore, right? Maybe we want to only save the resource we need per the config, and grab the other one if the config changes?

**Review by jcsackett — APPROVED** [review]
Looks good, with one optional comment.

If you don't want to do that now, please leave an XXX or TODO indicating it's a thing we can do to speed up install if we need to.

**Comment by makyo** [comment]
Thanks all. Creating a few bugs for investigation out of this

**Comment by makyo** [comment]
someone else willhave to merge

**Comment by jcsackett** [comment]
Who has write access to this repo?

**Comment by frankban** [comment]
Sorry, forgot to add the team as admin. Done now!

**Pull request merged**